travis.sh

```diff
@@ -2,12 +2,15 @@
 
 set -ex
 
-npm run lint
+yarn lint
 
 # install Olm so that we can run the crypto tests.
-npm install https://matrix.org/packages/npm/olm/olm-3.1.0-pre1.tgz
+# This will add Olm as dependency, since it's currently unlisted.
+# (`yarn` does not have an install dependency without adding mode.)
+# TODO: Should Olm be a listed dev dependency instead, so that we can have it for testing
+# and don't need to run an extra step here?
```

Comment:
this seems reasonable to me

Reply:
Thanks, it has been done: https://github.com/matrix-org/matrix-js-sdk/pull/856/commits/cc86f427d226241c7b5a6bde0346524faefca3fd